app/menu.js

```diff
@@ -1,8 +1,28 @@
 import { app, Menu, shell } from 'electron';
 import showMainWindow, { getMainWindow } from './mainWindow';
+const pkg = require('../package.json');
+const Path = require('path');
 
 export default class MenuBuilder {
   buildMenu() {
+
+    // https://www.electronjs.org/docs/all#appsetaboutpaneloptionsoptions
+    // version: MacOS only
+    // credits: MacOS & Windows only
+    // authors: Linux only
+    // website: Linux only
+    // iconPath: Linux & Windows only
+    app.setAboutPanelOptions({
+      applicationName: pkg.productName, 
+      applicationVersion: pkg.version,
+      copyright: `License: ${pkg.license}`,
+      version: pkg.version,
+      credits: pkg.author,
+      author: pkg.author,
+      website: pkg.homepage,
+      iconPath: Path.join(__dirname, '../resources/icons/128x128.png')
```

Comment:
@mslipper this may need to be tested on linux / windows specifically

Reply:
I think you should use this guy, coupled with `path.join`: https://www.electronjs.org/docs/api/process#processresourcespath-readonly

Reply:
Fixed in ffd8470d8e27ab7079aade315aeb0e7354166b4b

This works in the MacOS build (although I think it is actually ignored by MacOS, see https://www.electronjs.org/docs/api/app#appsetaboutpaneloptionsoptions)

I spent a day trying and failing to get Windows installed on a partition on my MacBook Pro so I still can't currently test on Windows. Can you check this works? I'll acquire some dedicated hardware for Window soon.

Reply:
I'll check this out when we build the next release. Merging for now.